### PR TITLE
Make dict file to be ZSTD compressed.

### PR DESCRIPTION
**Pull request by rocka**
Copied from https://github.com/fcitx/libime/compare/1b601d2...c70612a

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -23,8 +23,18 @@ include(ECMSetupVersion)
 include(ECMGenerateHeaders)
 include(ECMGeneratePkgConfigFile)
 include(ECMUninstallTarget)
+find_package(PkgConfig REQUIRED)
 
+pkg_check_modules(ZSTD REQUIRED IMPORTED_TARGET "libzstd")
 find_package(Fcitx5Utils REQUIRED)
+find_package(fmt REQUIRED)
+
+if (TARGET fmt::fmt-header-only)
```
Comment (wengxt) [inline CMakeLists.txt]: Why fmt is here? It seems no code using it.

**Review by wengxt — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (rocka) [reply CMakeLists.txt]: It's used in engine.cpp and missing in dependency ...

https://github.com/fcitx/libime-jyutping/blob/9d9dba3f22d0b9f09a639556f6d4c9a9c51e5c97/src/engine/engine.cpp#L29

https://github.com/fcitx/libime-jyutping/blob/9d9dba3f22d0b9f09a639556f6d4c9a9c51e5c97/src/engine/engine.cpp#L95

Comment (wengxt) [reply CMakeLists.txt]: Please submit a separate PR for that.

Also target_link_libraries need to be fixed too

Comment (rocka) [reply CMakeLists.txt]: https://github.com/fcitx/libime-jyutping/pull/8

**Pull request merged**